lang/rs/bindings/cne/examples/loopback/src/main.rs

```diff
@@ -0,0 +1,221 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright (c) 2020-2022 Intel Corporation.
+ */
+
+use clap::{App, Arg};
+use crossbeam_channel::{bounded, select, tick, Receiver};
+use ctrlc;
+use etherparse::Ethernet2Header;
+use std::sync::atomic::{AtomicBool, Ordering};
+use std::sync::Arc;
+use std::thread;
+use std::time::Duration;
+
+use cne_binding::error::*;
+use cne_binding::instance::*;
+use cne_binding::packet::*;
+use cne_binding::port::*;
+
+fn main() {
+    // Parse command line arguments.
+    let matches = App::new("loopback")
+        .version("0.1.0")
+        .about("CNE loopback example application.")
+        .arg(
+            Arg::with_name("config")
+                .short("c")
+                .long("config")
+                .required(true)
+                .takes_value(true)
+                .help("CNE JSON config file"),
+        )
+        .arg(
+            Arg::with_name("port")
+                .short("p")
+                .long("port")
+                .required(true)
+                .takes_value(true)
+                .help("port name in JSON file"),
```

Comment:
Did this one change to a number and not a name?

Reply:
correct. I'll update it to port index